app/views/users/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```

Comment:
Maybe since we get it during signup, we could use `@resource.first_name` here in place of `@email`?